### ISSUE TITLE
3D skin layer isn't rendered

### ISSUE THREAD

**Issue by ShaneTRS** [tr7zw/3d-Skin-Layers#9]
3D skin layer isn't rendered at all until you see yourself or another player.
It becomes visible when: you go into 3rd person, open your inventory, or your eyes meet with another player.

**Comment by tr7zw** [comment]
Dumb question: how do you see yourself when not in 3rd person/open inventory^^. If you mean the first person arm, the issue is, that the 3d model of the layer gets generated when a player gets rendered the first time while having a skin loaded. In first person the own player is not rendered, so the arm 3d model isn't yet generated. Guess I could look into adding a workaround for that.

**Comment by ShaneTRS** [comment]
Yeah, the only instance I can think of was the first-person arm. It's most definitely not a huge issue since you'll almost always open your inventory in some way. But in creative mode that can take a surprisingly long time, since there is only a player in the "survival inventory" tab.
